app/routes/admin/tenants.py

```diff
@@ -226,12 +287,32 @@ async def delete_tenant_by_id(
         bound_logger.debug("Successfully deleted tenant.")
 
 
-@router.get("/{wallet_id}/access-token", response_model=TenantAuth)
+@router.get(
```

Comment:
Note to self: this should really be a PATCH endpoint, so it's more clear that this modifies existing tokens.

Maybe we can mark this one as deprecated, and copy it identically with the one modified to being a `patch` endpoint

One of those things that should just be a done...